src/components/tables/Store.ts

```diff
@@ -9,11 +9,27 @@ import {
     getStoreWithHydrationSettings,
     StoreWithHydration,
 } from 'stores/Hydration';
-import { Schema } from 'types';
 import { devtoolsOptions } from 'utils/store-utils';
 import create, { StoreApi } from 'zustand';
 import { devtools, NamedSet } from 'zustand/middleware';
 
+export interface StatsSchema {
```

Comment:
This is here just for the tables to render. Eventually this will get moved into the same type that is used with the query.